src/core/Microsoft.PowerToys.Settings.UI.Lib/ImageResizerSettings.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerToys.Settings.UI.Lib
+{
+    public class ImageResizerSettings
+    {
+        [JsonPropertyName("version")]
+        public string Version { get; set; }
+
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        [JsonPropertyName("properties")]
+        public ImageResizerProperties Properties { get; set; }
+
+        public ImageResizerSettings()
+        {
+            this.Version = "1";
+            this.Name = "Image Resizer";
```

Comment:
In the old settings.json this was `ImageResizer`.

Reply:
Yeah, but on the previous version if you check the JSON writing code within the C# side it was saved as `ImageResizer` (I suppose that can be considered like a bug since initially the project was named ImageResizer but then the space was added later but it was not added in that json writing code). Not sure if that may cause any conflicts for users that were already using Image Resizer, since they would have the old json file like this. We should just make sure that switching to SettingsV2 should not be a breaking change and whatever settings they have used earlier should be readable. So maybe while writing we make sure we use the name `Image Resizer` but while loading it can check for either `Image Resizer` or `ImageResizer`?

This is how my old PowerToys/ImageResizer/settings.json file was structured.
` {"version":"1.0","name":"ImageResizer","properties":{"imageresizer_selectedSizeIndex":{"value":0},"imageresizer_shrinkOnly":{"value":false},"imageresizer_replace":{"value":false},"imageresizer_ignoreOrientation":{"value":false},"imageresizer_jpegQualityLevel":{"value":90},"imageresizer_pngInterlaceOption":{"value":0},"imageresizer_tiffCompressOption":{"value":0},"imageresizer_fileName":{"value":"%1 (%2)"},"imageresizer_sizes":{"value":[{"name":"Small","fit":1,"width":854.0,"height":480.0,"unit":3},{"name":"Medium","fit":1,"width":1366.0,"height":768.0,"unit":3},{"name":"Large","fit":1,"width":1920.0,"height":1080.0,"unit":3},{"name":"Phone","fit":1,"width":320.0,"height":568.0,"unit":3}]},"imageresizer_keepDateModified":{"value":true},"imageresizer_fallbackEncoder":{"value":"19e4a5aa-5662-4fc5-a0c0-1758028e1057"},"imageresizer_customSize":{"value":{"fit":2,"width":50.0,"height":700.0,"unit":2}}}}`